nntrainer/utils/util_func.cpp

```diff
@@ -92,21 +93,18 @@ Tensor zero_pad(int batch, Tensor const &in, unsigned int const *padding) {
 }
 
 // This is strip pad and return original tensor
-Tensor strip_pad(Tensor const &in, unsigned int const *padding,
-                 Tensor &output) {
+Tensor strip_pad(Tensor const &in, unsigned int const *padding, Tensor &output,
```

Comment:
Same. Alternatively, I guess we can return `Tensor &`

Reply:
done